src/components/DropDownMenu.tsx

```diff
@@ -111,7 +113,9 @@ export const DropDownMenu = () => {
               <Menu.Item>
                 <Button
                   icon="snow"
-                  content={showSnow ? "Skru av snø" : "Skru på snø"}
+                  content={(showSnow && !prefersReducedMotion) ? "Skru av snø" : "Skru på snø"}
+                  title={prefersReducedMotion ? "Animasjoner er satt til redusert i operativsystemet ditt. Skru av dette for å kunne se nydelig animert snø ☃️." : ""}
```

Comment:
Vi bruker `title` en del. Husk at `title` tilnærmet ikke eksisterer for skjermleser-brukere.

Reply:
Hmm. Det jeg egentlig tenkte var at det var nøyaktig i det tilfellet det var til hjelp med en presisering i title. Lite Google søk og jeg finner tre forskjellige svar på hva som er anbefalt å bruke 😅

Reply:
Hvis du skal gå all-in på UU er dette nesten en `aria-describedby` den mest "korrekte" løsningen her.

Reply:
Alternativt ser jeg ikke et kjempestort problem i å bruke `aria-label` og `title` og bare overstyre den faktiske knappeteksten. Alle brukere vil strengt tatt få mulighet til å få den samme informasjonen da.